ERROR - 2023-05-24 11:49:24 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#5 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-05-24 11:49:24 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `mahasiswa`
JOIN `prodi` ON `prodi`.`id_prodi`=`mahasiswa`.`prodi_mahasiswa`', [], false)
 3 APPPATH\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:24 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:49:24 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:32 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:49:32 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#5 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-05-24 11:49:34 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `mahasiswa`
JOIN `prodi` ON `prodi`.`id_prodi`=`mahasiswa`.`prodi_mahasiswa`', [], false)
 3 APPPATH\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:49:34 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:49:34 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:44 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\UTS_CI\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->add()
#10 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#11 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\UTS_CI\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->add()
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#10 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-05-24 11:49:44 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `prodi` (`nama_prodi`) VALUES (:nama_prodi:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->add()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:49:48 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\UTS_CI\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->add()
#10 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#11 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\UTS_CI\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->add()
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#10 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-05-24 11:49:48 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `prodi` (`nama_prodi`) VALUES (:nama_prodi:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\ProdiController.php(21): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->add()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:50:02 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:50:02 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:50:06 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\app\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
#5 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-05-24 11:50:06 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `mahasiswa`
JOIN `prodi` ON `prodi`.`id_prodi`=`mahasiswa`.`prodi_mahasiswa`', [], false)
 3 APPPATH\Models\MahasiswaModel.php(47): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\MahasiswaController.php(50): App\Models\MahasiswaModel->getJoinedMahasiswa()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetch()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:50:06 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:50:06 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\MahasiswaController.php(101): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\MahasiswaController->fetchOption()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MahasiswaController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 11:50:09 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#9 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\UTS_CI\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\UTS_CI\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\UTS_CI\app\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
#7 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
#8 C:\xampp\htdocs\UTS_CI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\UTS_CI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\UTS_CI\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-24 11:50:09 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci_uts'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `prodi`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProdiController.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProdiController->fetch()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProdiController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\UTS_CI\\public\\index.php')
ERROR - 2023-05-24 12:03:05 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ci_uts', 3306, '', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\UTS_CI\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\UTS_CI\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\UTS_CI\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\UTS_CI\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'ci_uts' in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\UTS_CI\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\UTS_CI\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\UTS_CI\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\UTS_CI\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-05-24 12:12:41 --> mysqli_sql_exception: Table 'prodi' already exists in C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\UTS_CI\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\UTS_CI\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\UTS_CI\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\UTS_CI\app\Database\Migrations\2023-05-15-123756_CreateProdiTable.php(24): CodeIgniter\Database\Forge->createTable('prodi')
#5 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProdiTable->up()
#6 C:\xampp\htdocs\UTS_CI\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\UTS_CI\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\UTS_CI\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\UTS_CI\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\UTS_CI\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
